cmd/ttn-lw-cli/commands/end_device_templates.go

```diff
@@ -347,9 +347,9 @@ mapping many end device templates with a generic template.
 Typical use cases are:
 
 1. Assigning identifiers from a mapping file to device templates matching on
-   mapping key.
+	mapping key.
```

Comment:
What’s the change here; spaces to tabs? That’s gonna break here; this is alignment for CLI